skimage/measure/fit.py

```diff
@@ -730,14 +928,24 @@ def ransac(
         ``is_model_valid(model, *random_data)`` and
         ``is_data_valid(*random_data)`` must all take each data array as
         separate arguments.
-    model_class : object
-        Object with the following object methods:
+    model_class : type
+        Class with the following methods:
 
-         * ``success = estimate(*data)``
-         * ``residuals(*data)``
+        * Either:
+
+          * ``tf = model_class.from_estimate(*data)`` (``from_estimate`` class
```

Comment:
Would start with ``from_estimate`` class method, and then give the description. Same the line below: ``estimate`` instance method: ...